orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java

```diff
@@ -353,6 +353,7 @@ private static EnsemblDataCache loadEnsemblDataCache(@NotNull OrangeConfig confi
     {
         EnsemblDataCache ensemblDataCache = new EnsemblDataCache(config.ensemblDataDirectory(),
                 RefGenomeVersion.from(config.refGenomeVersion().name()));
+        ensemblDataCache.setRequireNonEnsemblTranscripts();
```

Comment:
Would `includeNonEnsemblTranscripts()` be a better name for this method? Also, is the fix captured by any tests?

Reply:
The current function name is consistent with similar function names in this object and the object is part of hmf-common so I'd rather not touch that.

`OrangeAlgoTest` fails without this line, because it uses a dummy ensembl cache where none of the transcripts match with the ensembl pattern. Its not explicit but this failing test was the actual reason I found this bug in the first place so at least its tested!